R/s3.R

```diff
@@ -84,6 +86,14 @@ s3_put <- function(remote_ind, local_source,  mock_get=c('copy','move','none'),
   # identify the remote data file to be indicated by remote_ind
   data_file <- as_data_file(remote_ind, ind_ext=ind_ext)
   
+  # allow for dry runs of s3_put, where we move and create files locally but do
+  # nothing on S3
+  if(isTRUE(dry_put)) {
+    sc_indicate(ind_file=remote_ind, warning="dry_put=TRUE; not actually pushed", data_file=data_file)
+    mock_move_copy(mock_get, local_file, data_file)
+    return(NA)
```

Comment:
Looks like `s3_put` returns an `NA` and `gd_put()` returns `NULL`. Should they both be the same?